plugin-taskbar/lxqttaskbar.h

```diff
@@ -65,16 +65,17 @@ class LxQtTaskBar : public QFrame
 
     void realign();
 
-    Qt::ToolButtonStyle buttonStyle() { return mButtonStyle; }
-    int buttonWidth() { return mButtonWidth; }
-    bool closeOnMiddleClick() { return mCloseOnMiddleClick; }
-    bool raiseOnCurrentDesktop() { return mRaiseOnCurrentDesktop; }
-    bool isShowOnlyCurrentDesktopTasks() { return mShowOnlyCurrentDesktopTasks; }
-    bool isShowOnlyCurrentScreenTasks() { return mShowOnlyCurrentScreenTasks; }
-    bool isShowOnlyMinimizedTasks() { return mShowOnlyMinimizedTasks; }
-    bool isAutoRotate() { return mAutoRotate; }
-    bool isGroupingEnabled() { return mGroupingEnabled; }
-    bool isShowGroupOnHover() { return mShowGroupOnHover; }
+    Qt::ToolButtonStyle buttonStyle() const throw () { return mButtonStyle; }
```

Comment:
Just stating that this method never throws any exception ... in case it would throw in run-time, the process will be aborted.
(QT doesn't use exceptions for error handling and also LXQt component don't... it's my habbit to state simple getters as non-throwable...but I can rip it off)